cloudformation/security-hq.template.yaml

```diff
@@ -15,15 +15,15 @@ Metadata:
     - Label:
         default: Networking
       Parameters:
-      - VpcId
+      - OldVpcId
       - Subnets
       - TLSCertArn
 
 Parameters:
   SecurityHQSourceBundleBucket:
     Type: String
     Description: S3 bucket containing Security HQ's source bundle
-  VpcId:
+  OldVpcId:
```

Comment:
We ran into a similar problem [during the last migration fortnight](https://docs.google.com/document/d/1NKEUbua8XNA69gYwRCULV6bQ2E09d7fxtG8XHCxsqxs/edit?disco=AAAAOJcsGlk). Presumably this change means that we'll need to apply the update manually (i.e. not via Riff-Raff) in order to populate the new parameter value?

Reply:
Good point, this will require a manual initial deploy.

In this case, we have parameters with different types but also values - as the CDK version runs in a different VPC too.